sdk/anomalydetector/ai-anomaly-detector/package.json

```diff
@@ -71,7 +71,7 @@
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
-    "@azure/identity": "2.0.0-beta.4",
```

Comment:
Why arent we moving to beta.6 here?

Reply:
My bad, I will revisit all of these references asap.